MojioSDK/Client/Core/RestClient.swift

```diff
@@ -349,11 +345,15 @@ open class RestClient {
      public func query(top: String?, skip: String?, filter: String?, select: String?, orderby: String?) -> Self {
      return self.query(top, skip: skip, filter: filter, select: select, orderby: orderby, since: nil, before: nil, fields: nil)
      }*/
-    open func run(arrayEncoding: URLEncoding.ArrayEncoding = .noBrackets, completion: @escaping (_ response: Codable?, _ debugInfo: RestClientRequestDebugInfo?) -> Void, failure: @escaping (_ error: Any?) -> Void) {
-        self.run(arrayEncoding: arrayEncoding, completion: {response, headers, debugInfo in completion(response, debugInfo)}, failure: failure)
+    open func run(debug: ((_ request: Request?, _ response: DataResponse<Data>?, _ date: Date?) -> Void)? = nil,
```

Comment:
`response` contains `request` and not sure that we need the date
`response` contains `timeline` that contains time-related info about the lifecycle of the request

Reply:
@andriimakukha